scripts/drtprod

```diff
@@ -308,7 +283,7 @@ EOF"
         # setup dns
         $0 dns drt-chaos create
         ;;
-      "drt-ua1")
+      "ua")
```

Comment:
nit: should this be `drt-ua` ?

Reply:
shrug. It isn't a cluster name anymore, since it makes both clusters, so I'm not sure it matters?

Reply:
Sounds good. Merge away